.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: CI
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      BASE_IMAGE_VERSION: "3.8-alpine3.11"
+      POETRY_VERSION: "1.0.2"
+      TAG: "python3.8-poetry1.0"
+
+      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+      DOCKER_REGISTRY: moneymeets/ruby-python-awscli
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Build and publish image
+        run: |
+          if [ $GITHUB_REF != "refs/heads/master" ]; then export TAG=$TAG-${GITHUB_REF##*/} ; fi
```

```suggestion
          if [ "$GITHUB_REF" != "refs/heads/master" ]; then export TAG=$TAG-${GITHUB_REF##*/} ; fi
```

... either use quotes, or the old good `x` trick...